src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandler.java

```diff
@@ -477,7 +477,7 @@ private CallbackResponse submitClaim(CallbackParams callbackParams) {
         dataBuilder.featureToggleWA(toggleConfiguration.getFeatureToggle());
 
         //assign case management category to the case and caseNameHMCTSinternal
-        dataBuilder.caseNameHmctsInternal(buildCaseNameInternal(caseData));
+        dataBuilder.caseNameHmctsInternal(buildCaseName(caseData));
 
```

Comment:
dataBuilder.caseNamePublic(CaseNameUtils.buildCaseName(caseData));
this is missing in this file?

Reply:
will be fixed in here: https://tools.hmcts.net/jira/browse/CIV-12289